context.tf

```diff
@@ -145,7 +145,7 @@ variable "delimiter" {
 
 variable "attributes" {
   type        = list(string)
-  default     = []
+  default     = ["config"]
```

Comment:
Unfortunately, `context.tf` is a file that is automatically distributed and updated, so we cannot allow any changes to it.

You may, however, add a `config_attributes` variable along the same lines as the `iam_attributes` variable you added and work around the issue that way.